bmcremedy_connector.py

```diff
@@ -490,12 +630,14 @@ def _make_rest_call(self, endpoint, action_result, headers=None, params=None, da
             return RetVal3(action_result.set_status(phantom.APP_ERROR, error_message), response_data, response)
 
         try:
+            # BMC SDK want X-Requested-By header in every request for Protection against CSRF attack
+            headers["X-Requested-By"] = "XMLHttpRequest"
             if files:
                 response = request_func('{}{}'.format(self._base_url, endpoint), headers=headers, files=files,
                                     verify=self._verify_server_cert, timeout=consts.BMCREMEDY_DEFAULT_TIMEOUT)
             else:
                 response = request_func('{}{}'.format(self._base_url, endpoint), headers=headers, data=data, params=params,
-                                        verify=self._verify_server_cert, timeout=consts.BMCREMEDY_DEFAULT_TIMEOUT)
```

Comment:
Why remove the timeout?

Reply:
'list ticket' action is taking more time to fetch data from BMC server. Therefore I have removed the timeout key.

Reply:
Maybe we can increase the timeout, but completely removing it sounds like a SOARHELP in the making once someone's playbook hangs